### PR TITLE
Move EOA from contracts page to governance overview

### LINKED ISSUE #177: Move EOA addresses from Contracts page
### Description

1. Move EOA addresses from the Contracts page to the Governance overview page in the user docs.
2. Add brief info about each address

### Location
`building-on-lisk/contracts.mdx`
`docs/docs-user/governance/overview.mdx`

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
### What was the problem?

This PR resolves #177

### CODE REVIEW

File: docs/docs-user/governance/overview.mdx [hunk 1]
```diff
@@ -141,3 +141,65 @@ The foundation will have an administrative role and will take care of the follow
...
+      <td>🟢</td>
+    </tr>
+    <tr>
+    <td>[0xF54791059df4a12BA461b881B4080Ae81a1d0AC0](https://etherscan.io/address/0xF54791059df4a12BA461b881B4080Ae81a1d0AC0)</td>
+    <td>🟡</td>
+    </tr>
+
+    <tr>
+      <th scope="row" rowSpan="2">Batcher</th>
+      <th scope="row" rowSpan="2">Managed by Gelato</th>
```
Comment (Tschakki) [inline docs/docs-user/governance/overview.mdx]: What is the difference between "Managed by Gelato" and "The contract is owned by a multi-sig account that belongs to Gelato."?

File: docs/docs-user/governance/overview.mdx [hunk 2]
```diff
@@ -141,3 +141,65 @@ The foundation will have an administrative role and will take care of the follow
...
+    </tr>
+
+    <tr>
+      <th scope="row" rowSpan="2">SystemConfig</th>
+      <th scope="row" rowSpan="2">The contract is owned by a multi-sig account that belongs to Gelato.</th>
+      <td>[0x05f23282FFDCA8286E4738C1aF79079f3d843750](https://etherscan.io/address/0x05f23282FFDCA8286E4738C1aF79079f3d843750)</td>
+      <td>🟢</td>
+    </tr>
+    <tr>
+    <td>[0xF54791059df4a12BA461b881B4080Ae81a1d0AC0](https://etherscan.io/address/0xF54791059df4a12BA461b881B4080Ae81a1d0AC0)</td>
```
Comment (Tschakki) [inline docs/docs-user/governance/overview.mdx]:
```suggestion
    <td>[0xF54791059df4a12BA461b881B4080Ae81a1d0AC0](https://sepolia.etherscan.io/address/0xF54791059df4a12BA461b881B4080Ae81a1d0AC0)</td>
```

**Review by Tschakki — CHANGES_REQUESTED** [review]

File: docs/docs-user/governance/overview.mdx (hunk 1 above)
Comment (TalhaMaliktz) [reply docs/docs-user/governance/overview.mdx]: This is what Gelato had to say on this topic!

> The multi-sig account is in Gelato ownership. as part of the Superchain agreement, the ownership of the multisig will be transferred to a 2/2 multisig controlled by the Security Council and Optimism Foundation later on. The process for transfer the ownership isn't yet established.

Comment (TalhaMaliktz) [reply docs/docs-user/governance/overview.mdx]: For more information, see here for [EOA vs Contract](https://blog.ambire.com/eoas-vs-smart-contract-accounts/) ownership

Comment (Tschakki) [reply docs/docs-user/governance/overview.mdx]: hmm ok, so both are managed by gelato, but difference is that some or multi-sig and some not? 
Imho for users, it is most relevant who manages the account, so I would suggest to change the column "Type of Key" to "Controlled by" or "Owner". So currently we can simply write "Gelato" in this column for all the EOAs. Once Gelato updates the ownership, we can simply update the table accordingly

**Review by Tschakki — APPROVED** [review]
LGTM

**Pull request merged**